### PR TITLE
docs/reference, how-to: add store limit topics

### PR DESCRIPTION
**Pull request by dcalvin**
## What have you changed? (mandatory)

Please explain **IN DETAIL** what the changes are in this PR and why they are needed:

- This PR added 
    - a how-to topic: configuring the scheduling rate limit  
   -  a reference topic: pd-control

## What are the type of the changes? (mandatory)

- Doc updates

### CODE REVIEW

File: docs/how-to/configure/store-limit.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    > The modification immediately only takes effect on newly added stores, and will be applied to the entire cluster after TiKV restarts. If you want to apply speed limits on all stores or an individual store, consider using the second method below.
+
+- Use the `pd-ctl` tool to view or modify the upper limit of the scheduling rate. The commands are:
+
+    - **`stores show limit`**
+        
+        Example:
+
+        ```bash
+        » stores show  // If store-balance-rate is set to 15, the corresponding rate for all stores should be 15 / 60 = 0.25.
```
Comment (Hoverbear) [inline docs/how-to/configure/store-limit.md]:
```suggestion
        » stores show limit // If store-balance-rate is set to 15, the corresponding rate for all stores should be 15 / 60 = 0.25.
```

File: docs/how-to/configure/store-limit.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ PD provides the following two methods to configure scheduling rate limits on sto
 
     > **Note:**
     >
-    > The modification immediately only takes effect on newly added stores, and will be applied to the entire cluster after TiKV restarts. If you want to apply speed limits on all stores or an individual store, consider using the second method below.
+    > The modification only takes effect on stores added after this configuration change, and will be applied to all stores in the  cluster after you restart TiKV. If you want this change to work  immediately on all stores or some individual stores before the change without restarting, combine this configuration with the following method. See [Sample usages](#sample-usages) for more details.
```
Comment (Hoverbear) [inline docs/how-to/configure/store-limit.md]: Can we be more specific about which following method?

Comment (dcalvin) [reply docs/how-to/configure/store-limit.md]: Good point. updated. PTAL again.

File: docs/how-to/configure/store-limit.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+    Example:
+
+    ```bash
+    >> config set store-balance-rate 20
+    ```
+
+    > **Note:**
+    >
+    > The modification only takes effect on stores added after this configuration change, and will be applied to all stores in the  cluster after you restart TiKV. If you want this change to work  immediately on all stores or some individual stores before the change without restarting, combine this configuration with the `pd-ctl` tool method below. See [Sample usages](#sample-usages) for more details.
```
Comment (rleungx) [inline docs/how-to/configure/store-limit.md]:
```suggestion
    > The modification only takes effect on stores added after this configuration change, and will be applied to all stores in the cluster after you restart TiKV. If you want this change to work immediately on all stores or some individual stores before the change without restarting, combine this configuration with the `pd-ctl` tool method below. See [Sample usages](#sample-usages) for more details.
```

File: docs/how-to/configure/store-limit.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+In TiKV, PD generates different scheduling operators based on the information gathered from TiKV and scheduling strategies. The operators are then sent to TiKV to perform scheduling on Regions. You can use `*-schedule-limit` to set speed limits on different operators, but this may cause performance bottlenecks in certain scenarios because these parameters function globally on the entire cluster. Rate limit at the store level allows you to control scheduling more flexibly with more refined granularities.
+
+## How to configure scheduling rate limits on stores
+
+PD provides the following two methods to configure scheduling rate limits on stores:
+
+- Configure the rate limit using **`store-balance-rate`**.
+
+    `store-balance-rate` specifies the maximum number of scheduling tasks allowed for each store per minute. The scheduling steps include adding Peers or Learners. Set this parameter in the PD configuration file. The default value is 15. This configuration is persistent.
```
Comment (rleungx) [inline docs/how-to/configure/store-limit.md]:
```suggestion
    `store-balance-rate` specifies the maximum number of scheduling tasks allowed for each store per minute. The scheduling steps include adding peers or learners. Set this parameter in the PD configuration file. The default value is 15. This configuration is persistent.
```

Comment (dcalvin) [reply docs/how-to/configure/store-limit.md]: PTAL again. @rleungx

**Comment by dcalvin** [comment]
@Hoverbear PTAL and merge.  Thanks.

**Pull request merged**